src/main/java/az/gdg/msarticle/util/MailUtil.java

```diff
@@ -0,0 +1,40 @@
+package az.gdg.msarticle.util;
+
+import az.gdg.msarticle.model.client.mail.MailDTO;
+import az.gdg.msarticle.service.MailService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MailUtil {
+    private static final Logger logger = LoggerFactory.getLogger(MailUtil.class);
+
+    private MailUtil() {
+    }
+
+    public static MailDTO buildMail(String mailBody) {
+        List<String> receivers = new ArrayList<>();
+        receivers.add("gdg.rubber.duck@gmail.com");
+        receivers.add("movsum.nigar@gmail.com");
+        receivers.add("asif.hajiyev@outlook.com");
+        receivers.add("huseynov_ali@outlook.com");
+        receivers.add("isgandarli_murad@mail.ru");
```

Comment:
it should not be hardcoded, because admin list can change. Admins should be request from db in non-util class and passed as a parameter.